include/boost/geometry/algorithms/detail/within/within_no_turns.hpp

```diff
@@ -18,6 +18,8 @@
 #ifndef BOOST_GEOMETRY_ALGORITHMS_DETAIL_WITHIN_WITHIN_NO_TURNS_HPP
 #define BOOST_GEOMETRY_ALGORITHMS_DETAIL_WITHIN_WITHIN_NO_TURNS_HPP
 
+#include <boost/type_traits/is_base_of.hpp>
```

Comment:
Sorry, I had not seen this PR before.
And I tackled the same problem because I got a mail that my merge was red.

I think this  line is not the right fix - we use it from `std` now (in several other places)